zh_cn/02.安装与配置/03.helm.md

```diff
@@ -47,12 +48,14 @@ helm upgrade --install zadig --namespace zadig  koderover-chart/zadig --set endp
 > IP 为 Kubernetes 任一节点的外网可访问 IP。
 
 ```bash
-helm upgrade --install zadig --namespace zadig  koderover-chart/zadig --set endpoint.type=IP \
+helm upgrade --install zadig --namespace <NAMESPACE> koderover-chart/zadig --set endpoint.type=IP \
     --set endpoint.IP=<IP> \
     --set ingress-nginx.controller.service.nodePorts.http=<30000~32767任一端口>
-# v1.7.0+ 版本需加上以下参数
+# v1.7.0+ 版本需加上以下参数，且确保 global.extensions.extAuth.extauthzServerRef.namespace 和安装 Zadig 系统的 Namespace 一致
```

Comment:
The following parameters need to be added: `--set dex.fullnameOverride=<IP>:<PORT>`

Reply:
Same for this. It better be not related to the endpoint but related to the namespace